docs/servo_power_monitor/src/index.ts

```diff
@@ -181,7 +187,7 @@ function setupStartUSBButton() {
       });
     } catch (err) {
       console.error(`Disconnected: ${err}`);
-      device = null;
+      // device = null;
       requestUSBButton.disabled = false;
```

Comment:
When an error is caught here, nothing is returned in the current implementation because closing usb port without error. But the argument of readLoop is Promise, so TypeScript says the error Argument of type '() => Promise<string | undefined>' is not assignable to parameter of type '() => Promise<string>'. .

Reply:
null and undefined are not the same. You can assign it to undefined and the type check should get through.